### DISCUSSION TITLE
Please add standalone cli as a package to Chocolatey

### DISCUSSION THREAD

**Discussion by tesar-tech** [tailwindlabs/tailwindcss#6650] [Ideas]
[Chocolate](https://community.chocolatey.org/) is a great package manager used among many windows developers.
Please add support for installing tailwindcss cli using choco.

Whole installation process would reduce into `cinst tailwindcss` command.

**Comment by mariposaben** [comment]
Totally agree with @tesar-tech . Please get it added for us. Thank you.

**Comment by tesar-tech** [comment]
You still cant install tailwind through choco, but you can do it through winget:

> winget install -e --id TailwindLabs.TailwindCSS

**Comment by ddjerqq** [comment]
please add it!
